blazor/scheduler/views.md

```diff
@@ -155,21 +162,30 @@ Usually a day view displays a single day with all its related appointments. It i
 
 N> All the above defined properties can be accessed within Day view except `AllowVirtualScrolling` and `HeaderRows`.
 
+![Displaying Day View in Blazor Scheduler](images\blazor-scheduler-views-day.png)
```

Comment:
invalid path reference.
images/blazor-scheduler-views-day.png

Reply:
resolve the issue  in path reference